### ISSUE TITLE
Support for 12D TIN

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/MDAL#163]
Some info here:
http://12dwiki.com.au/retriangulate-tin/

Examples and format spec: UNKNOWN!

**Comment by nyalldawson** [comment]
There's a mix of possible formats.

See
.12da
https://www.12d.com/downloads/pdf/12d_archive_file_format.pdf

.12daz : zipped 12da file

.12dxml: http://downloads.12dmodel.com/v11/11.1.13/11.1.13.1/12d_XML_File_Format_18_Nov_16.pdf

At least the formats are well documented...

**Comment by HAOOFE** [comment]
Yep.. 12da format has been used by surveyors in Australia very widely.. Please add support for it.. and make a feature to export all the supported format converting into DXF.

**Comment by HAOOFE** [comment]
Just upload two example. There are 12da files I received from client, and the corresponding dxf files I got my friend helped converted, He has a 12d software.
https://www.dropbox.com/sh/1q7o7d5pqtwbljt/AABaXtCaWvWhtfYXPUhB1Ncva?dl=0
hope them help.

**Comment by saberraz** [comment]
@HAOOFE Thanks for the example files. As this is a closed source format and we have not much info on the spec, it would be good if someone from 12d Solutions can provide us with the detailed spec and ideally fund the work!
